### PR TITLE
Fixes #3 - changed app configuration to get variables from the environment

### LINKED ISSUE #3: Update app configuration to get the dev and prod configuration from the environment
Currently, there are variables that can't be set at build time (e.g. proxyUser) that differ by server. These should come from the build environment so dev and production images can be built automatically.

### PR DESCRIPTION
**Pull request by navarroc**
This allows injection of environment variables at build time to the Docker image so they can be picked up by the application. To test this, you can run the following command:

```
docker build --no-cache --build-arg REACT_APP_ENV="${REACT_APP_ENV}" --build-arg REACT_APP_DEMO_USER="${REACT_APP_DEMO_USER}" --build-arg REACT_APP_DEMO_PASSWORD="${REACT_APP_DEMO_PASSWORD}" --build-arg REACT_APP_DW_USER_ID="${REACT_APP_DW_USER_ID}" --build-arg REACT_APP_KEYCLOAK_URL="${REACT_APP_KEYCLOAK_URL}" --progress=plain --tag farmdoc/frontend .
```

If you set environment variables for those parameters they should be in the environment.

### CODE REVIEW

**Comment by navarroc** [comment]
I build this branch locally and tagged it PR-5. It's currently deployed on the develop server for testing.

File: src/app.messages.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const loginMessage = "Please login with your account to access the farmdo
 	" an account yet, use the 'REGISTER' button to create one. If you are having trouble receiving the verification" +
 	" email in your inbox, please check your spam/junk folder.";
 export const sessionExpired = "Your session has expired. Please login again to access the farmdoc decision tools";
-export const nassYieldsDisclaimer = "2019 and 2020 numbers are actual values of ARC/PLC payments. All 2019 and 2020 yields and prices are final.";
+export const nassYieldsDisclaimer = "Values are actual for 2020 through 2022. Projections are made for 2023 and 2024.";
```
Comment (ywkim312) [inline src/app.messages.js:67]: Can we make this string as variable as well? If so, we don't need to rebuild or update the code when the year changes? If not, you can ignore this comment

Comment (navarroc) [reply src/app.messages.js:67]: Can we leave it for now? My JS skills are minimal and I have spent too much time on this. We can open an issue maybe to see what improvements could be made and see if they want to address them later.

Comment (ywkim312) [reply src/app.messages.js:67]: That is okay for me. My JS skill is even way lower than yours. I will approve.

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**